typescript/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I know nothing about typescript: do we need to track external dependencies like this though? For other languages, you can do most things through `BUILD` or `WORKSPACE` files, so I'm surprised we need these.

Reply:
According to the documentation, we need those files as bazel builds entire projects. I have checked it works as it translated the code into a `.js` file in `bazel-bin` and when you execute it show hello world. I think we will improve this an all other examples for a most robust testing. This `json` file and more will increase in its dependency tree. Anyways, I will wait today, maybe we will have more comments from Will and Sara.

Just for your information, these are the rules I am using: https://github.com/aspect-build/rules_ts